part-spec/semiconductor.json

```diff
@@ -464,6 +467,139 @@
+                    "$comment": "units of amps",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "collectorBaseVoltage": {
+                    "description": "maximum voltage between collector and base terminals of BJT with an open emitter terminal (V_CBO)",
+                    "$comment": "units of volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "collectorEmitterVoltage": {
+                    "description": "maximum voltage between collector and emitter terminals of BJT with an open base terminal (V_CEO)",
+                    "$comment": "units of volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "emitterBaseVoltage": {
+                    "description": "maximum voltage between emitter and base terminals of BJT with an open collector terminal (V_EBO)",
+                    "$comment": "units of volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "totalPowerDissipation": {
+                    "description": "maximum power that can be continously dissipated under temperature conditions",
+                    "$comment": "units of watts",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "collectorBaseCutOffCurrent": {
+                    "description": "current into the collector terminal when the BJT's base and collector are reverse biased and the emitter is open (I_CBO)",
+                    "$comment": "units of amps",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "emitterBaseCutOffCurrent": {
+                    "description": "current into the base terminal when the BJT's base and emitter are reverse biased and the collector is open (I_EBO)",
+                    "$comment": "units of amps",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "dcGain": {
+                    "description": "ratio of collector current to base current (hfe)",
+                    "$comment": "units of number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "collectorEmitterSaturationVoltage": {
+                    "description": "collector-emitter voltage below with a change in base current does not impact collector current (VCE_SAT)",
+                    "$comment": "units of volts",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "baseEmitterSaturationVoltage": {
+                    "description": "base-emitter voltage required to ensure the collector is forward biased for certain current conditions (VBE_SAT)",
+                    "$comment": "units of volts",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
```

Comment:
We may also want to add the Collector-Emitter and the Collector-Base breakdown voltages.